src/risedevtool/src/service_config.rs

```diff
@@ -281,6 +281,18 @@ pub struct RedisConfig {
     pub address: String,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+#[serde(deny_unknown_fields)]
+pub struct ConnectorNodeConfig {
+    #[serde(rename = "use")]
+    phantom_use: Option<String>,
+    pub id: String,
+    pub port: u16,
+    pub address: String,
+    pub user_managed: bool,
```

Comment:
Remove this?

Reply:
I think it is an optional component for now.

Reply:
user_managed means that whether users will need to manually start them. It seems that we didn't use this property in ConnectorService...?

Reply:
Got it. Thanks